projects/common-components/src/lib/talks/talk-item/talk-item.component.scss

```diff
@@ -0,0 +1,73 @@
+.jsmu-talk-item {
+	display: flex;
+	flex-direction: column;
+	max-width: 65rem;
+	margin: 0 auto;
+
+	&__wrapper {
+		display: flex;
+		justify-content: center;
+	}
+
+	&__btn {
+		align-self: flex-end;
+		margin: 10px 45px 40px 0;
+	}
+
+	&__photo {
+		margin-top: 12px;
+
+		&-img {
```

Comment:
if __photo-img separate element please move it out 
otherwise it looks like part of another element

the same about mentee-link

Reply:
done